configure/RULES_BUILD

```diff
@@ -167,6 +167,8 @@ build: $(OBJSNAME) $(LIBTARGETS) $(PRODTARGETS) $(TESTPRODTARGETS) \
 inc: $(COMMON_INC) $(INSTALL_INC) $(INSTALL_CONFIGS) $(INSTALLS_CFG) \
 	$(INSTALL_HTMLS) $(INSTALLS_PERL_MODULES) $(INSTALL_SCRIPTS)
 
+doc: inc $(INSTALL_MARKDOWNS)
```

Comment:
How much extra time does generating the `$(INSTALL_MARKDOWNS)` files add to the length of a build? If it isn't very much I'd be inclined to drop the new `doc` action type, move `$(INSTALL_DOCS)` from `buildInstall` to `inc` and use that to install the generated markdown files (you would need to add an `$(INSTALL_DOC)` rule from `$(COMMON_DIR)` though).

Reply:
I am a bit rusty on the details, but unless I am missing some details, this rule is about markdown files that are auto-generated from .dbd.pod during the build. Those markdown files are of interest to the RTD build only, and when RTD build is done, they are scrapped - as they should IMHO. Installation elsewhere "permanently" does not add any value, if that is what is meant with this change.
@minijackson should correct if I got that wrong.

Reply:
Hi @ttkorhonen I believe you're right, but what I'm suggesting is that those files could be useful for developers too. It wouldn't add much time to the build if we were to run the pod-to-markdown conversion from the existing `make inc` target, and install the markdown files into the existing `$(TOP)/doc` where they're readable as a text file for anyone who doesn't want to open a browser to read the HTML versions.

I'm trying to avoid adding the additional `doc` target and the new install rules and directory by reusing an existing target, set of rules and install directory for these generated files. I have this branch checked out, if I can find enough time on Thursday or Friday this week (maybe on the plane home Friday pm) I will try making the changes as I see them.

Reply:
OK, that is of course fair enough. My only worry is that we try to achieve too much in one go, as I already wrote in other comments.

Reply:
> I'm trying to avoid adding the additional doc target and the new install rules and directory by reusing an existing target, set of rules and install directory for these generated files. I have this branch checked out, if I can find enough time on Thursday or Friday this week (maybe on the plane home Friday pm) I will try making the changes as I see them.

Do you want to do these changes in this PR or in a follow-up PR?

Note: if you try to install the markdown files in `$(TOP)/doc` only, they won't be included in the RTD build. IIRC, RTD refuses to include external files, i.e. files that are not in its root directory, which is `$(TOP)/documentation` in this case.